src/sentry/search/events/fields.py

```diff
@@ -1955,7 +1990,9 @@ def resolve_field(self, field: str) -> Union[SnqlColumn, SnqlFunction]:
         if match:
             raise NotImplementedError(f"{field} not implemented in snql field parsing yet")
 
-        if field in FIELD_ALIASES:
+        if field in FIELD_ALIASES or field in SNQL_FIELD_ALIASES:
```

Comment:
There's some small deviations between `FIELD_ALIASES` and `SNQL_FIELD_ALIASES` because previously, `issue.id` isn't part of `FIELD_ALIASES` and we would just query for `group.id` and rename the column in post processing.

The change here is that `SNQL_FIELD_ALIASES` is aliasing it to `issue.id` so the renaming in the post processing is no longer necessary. Though once aliases are supported in columns, we should be able to remove `issue.id` from `SNQL_FIELD_ALIASES` and simply add an alias to the generated column.